diff --git a/src/routes.js b/src/routes.js
index 11343be..62c13c9 100644
--- a/src/routes.js
+++ b/src/routes.js
@@ -9,7 +9,6 @@
 */
 
 import {
-  documentationTopicName,
   notFoundRouteName,
   serverErrorRouteName,
 } from 'docc-render/constants/router';
@@ -31,25 +30,34 @@ export const fallbackRoutes = [
 
 export const pagesRoutes = [
   {
-    path: '/tutorials/:id',
-    name: 'tutorials-overview',
+    path: '/live/tutorials-overview',
+    name: 'live-tutorials-overview',
     component: () => import(
       /* webpackChunkName: "tutorials-overview" */ 'theme/views/TutorialsOverview.vue'
     ),
+    meta: {
+      skipFetchingData: true,
+    },
   },
   {
-    path: '/tutorials/:id/*',
-    name: 'topic',
+    path: '/live/tutorials',
+    name: 'live-tutorials',
     component: () => import(
       /* webpackChunkName: "topic" */ 'theme/views/Topic.vue'
     ),
+    meta: {
+      skipFetchingData: true,
+    },
   },
   {
-    path: '/documentation*',
-    name: documentationTopicName,
+    path: '/live/documentation',
+    name: 'live-documentation',
     component: () => import(
       /* webpackChunkName: "documentation-topic" */ 'theme/views/DocumentationTopic.vue'
     ),
+    meta: {
+      skipFetchingData: true,
+    },
   },
 ];
 
